partiql-ast/src/main/kotlin/org/partiql/ast/sql/SqlLayout.kt

```diff
@@ -1,97 +1,78 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
 package org.partiql.ast.sql
 
 /**
  * [SqlLayout] determines how an [SqlBlock] tree is transformed in SQL text.
  */
-@Deprecated("This will be changed in the next major version")
```

Comment:
Given that we're changing the `SqlBlock`, `SqlDialect`, and `SqlLayout` public APIs, I think we should include an entry in the CHANGELOG for the 0.15 release.

Reply:
ack